AddNewCard.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+require_once 'src/Models/PlayerCardModel.php';
+
+$db = new PDO('mysql:host=db; dbname=Collection', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+if (isset($_POST['submit'])) {
+    $PlayerName = $_POST['PlayerName'];
+    $Club = $_POST['Club'];
+    $Position = $_POST['Position'];
+    $Defence = $_POST['Defence'];
+    $Control = $_POST['Control'];
+    $Attack = $_POST['Attack'];
+    $Player = new PlayerCardModel($db);
+    $newPlayer = $Player->addNewCard(
+        $PlayerName,
+        $Club,
+        $Position,
+        $Defence,
+        $Control,
+        $Attack
+    );
```

Comment:
You should use your PlayerCard entity here, pass the data into one of those and then give that to the model

Reply:
You should also do some basic validation here, making sure that the fields aren't empty, and that the data isn't too long for your database table